testing/scenario_app/bin/utils/adb_logcat_filtering.dart

```diff
@@ -74,6 +76,26 @@ extension type const AdbLogLine._(Match _match) {
     return null;
   }
 
+  @visibleForTesting
+  static const Set<String> kKnownNoiseTags = <String>{
+    'MonitoringInstr',
+    'ResourceExtractor',
+    'THREAD_STATE',
```

Comment:
nit: Is this really a noise tag? This gets dumped if a test hangs and tells you what the threads are doing right? Is it used otherwise.

Reply:
👍🏼 I moved this from `kKnownNoise` to `kKnownUsefulError`.